fab/services/templates/supervisor_formsplayer_spring.conf

```diff
@@ -0,0 +1,8 @@
+[program:{{ project }}-{{ environment }}-formsplayer-spring]
+command=java -jar {{ code_current}}/submodules/formsplayer/build/libs/formsplayer.jar
+user={{ sudo_user }}
+autostart=true
+autorestart=true
+stdout_logfile={{ log_dir }}/formsplayer-spring.log
+redirect_stderr=true
+stderr_logfile={{ log_dir }}/formsplayer-spring.error.log
```

Comment:
:+1: for startsecs. i think the `.jar` itself sets the port? or at least the default one. may be good to explicitly set it though so when people read this they know where the entry point is